stable/weblate/values.yaml

```
@@ -0,0 +1,39 @@
+deployment:
+  replicas: 1
+  image: weblate/weblate
+  tag: 4.2-1
+  container_port: 8080
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 0
+      maxSurge: 1
+  resources:
+    requests:
+      cpu: 300m
+      memory: 2048Mi
+    limits:
+      cpu: 500m
+      memory: 4096Mi
+  environment:
+    WEBLATE_DEBUG: False
+    WEBLATE_LOGLEVEL: WARNING
+    WEBLATE_SITE_TITLE: Weblate
+    WEBLATE_ADMIN_NAME: "Weblate Admin"
+    WEBLATE_ENABLE_HTTPS: True
+    SECURE_SSL_REDIRECT: True
+
+database:
+  create: true
```

This will error if the user doesn't have the postgres-controller installed so best disable by default it I think:

```suggestion
database:
  create: false
```